src/sentry/tasks/post_process.py

```diff
@@ -338,32 +355,37 @@ def handle_group_owners(project, group, issue_owners):
 
             # Owners already in the database that we'll keep
             keeping_owners = set()
-            for owner in current_group_owners:
-                owner_type = (
+            for group_owner in current_group_owners:
```

Comment:
renamed because mypy was confusing this `owner` with the `owner` on line 349 (and unfortunately they have different types)